ui/admin/app/templates/scopes/scope/host-catalogs/index.hbs

```diff
@@ -14,103 +14,125 @@
       {{t 'resources.host-catalog.title_plural'}}
       <DocLink @doc='host-catalog' @iconSize='large' />
     </h2>
-    {{#if @model}}
+    {{#if @model.hostCatalogsExist}}
       <p>{{t 'resources.host-catalog.description'}}</p>
     {{/if}}
   </page.header>
 
   <page.actions>
     {{#if (can 'create model' this.scope collection='host-catalogs')}}
-      {{#if @model}}
-        <Rose::LinkButton
+      {{#if @model.hostCatalogs}}
```

Comment:
change from `@model.hostCatalogs` to `@model.hostCatalogsExist`

Reply:
Fixed